### PR TITLE
Laravel 9

### PR DESCRIPTION
**Pull request by qschmick**
- Updated to allow Laravel 9 usage
- Updated CI to do a matrix of versions

### CODE REVIEW

**Comment by iadambrown** [comment]
Any idea when this might get merged?

**Comment by albanh** [comment]
Any chance we could have minor version bump in the mean time on this one please, it's the only thing holding up our 9 upgrade? 🙏🏻

**Comment by elfeffe** [comment]
@qschmick Any idea about when will be merged?

**Comment by qschmick** [comment]
@elfeffe when I get it all passing (tests and styling)

**Comment by albanh** [comment]
@qschmick does this now sort tests and styling? https://github.com/codestudiohq/laravel-totem/pull/327.  Keen to get this sorted as this is the only package we have holding us back for Laravel 9 and would be a real shame to have to drop it as it's really useful.  Cheers

**Comment by SwithFr** [comment]
Any update ?

**Comment by javis** [comment]
+1

File: src/Task.php [hunk 1]
```diff
@@ -56,7 +58,7 @@ class Task extends TotemModel
      *
      * @return bool
      */
-    public function getActivatedAttribute()
+    public function getActivatedAttribute(): bool
```
Comment (InfluxOW) [inline src/Task.php:61]: I have error `Studio\Totem\Task::getActivatedAttribute(): Return value must be of type bool, null returned` after trying to add a new task. When created, Task model doesn't have property `is_active` set (it's being set by database automatically as column has default value `true`). So, as an option, that method body should be `return $this->is_active ?? true`. Or we should refresh model in the repository (after [that](https://github.com/codestudiohq/laravel-totem/pull/314/files#diff-29ce9111f3d38f1c329e6c5936f7717392c41bc6b140adb2ee8bc459761e5539R100) line). Or smth else. Anyway, it's bugged right now.

[**Here's the full log**](https://privnote.com/UhVQAluo#qyQS7quT3).

**Comment by qschmick** [comment]
@iadambrown @albanh @InfluxOW @javis @SwithFr @elfeffe 
New 9.x release, sorry for the long delay, work and home life has been crazy. Thanks all those that helped and reviewed changes.

https://github.com/codestudiohq/laravel-totem/releases/tag/v9.0.0

**Pull request merged**